ush/config.community.sh

```diff
@@ -8,27 +8,20 @@ RUN_ENVIR="community"
 PREEXISTING_DIR_METHOD="rename"
 
 PREDEF_GRID_NAME="RRFS_CONUS_25km"
+GRID_GEN_METHOD="ESGgrid"
 QUILTING="TRUE"
-
-CCPP_PHYS_SUITE="FV3_GFS_v15p2"
-FCST_LEN_HRS="48"
+CCPP_PHYS_SUITE="FV3_GFS_2017_gfdlmp"
+FCST_LEN_HRS="6"
 LBC_SPEC_INTVL_HRS="6"
 
-DATE_FIRST_CYCL="20190615"
-DATE_LAST_CYCL="20190615"
+DATE_FIRST_CYCL="20190701"
```

Comment:
We need to keep these dates as the 20190615 date for the repo (that is the agreed upon default)